cpp/src/groupby/sort/quantiles/quantiles.cu

```diff
@@ -0,0 +1,225 @@
+/*

```

Comment:
This file is actually for the series quantile and not the group version. So it should be back in the quantiles directory. Similarly for the quantiles.hpp file.